localstack/services/dynamodb/dynamodb_listener.py

```diff
@@ -136,6 +137,43 @@ def return_response(self, method, path, data, headers, response):
             if 'StreamSpecification' in data:
                 create_dynamodb_stream(data)
             return
+        elif action == '%s.UpdateTimeToLive' % ACTION_PREFIX:
```

Comment:
I just realized in the travis-ci build output that we have a bunch of stacktraces like this one:
```
.17:00:29.013 [qtp2109798150-15] ERROR com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler - Unknown error
java.lang.ArrayIndexOutOfBoundsException: 1
at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.getAccessKey(LocalDynamoDBServerHandler.java:287) ~[DynamoDBLocal.jar:?]
at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.packageDynamoDBResponse(LocalDynamoDBServerHandler.java:307) ~[DynamoDBLocal.jar:?]
at com.amazonaws.services.dynamodbv2.local.server.LocalDynamoDBServerHandler.handle(LocalDynamoDBServerHandler.java:420) [DynamoDBLocal.jar:?]
```
This happens because we added the TTL logic in the `return_response(..)` method, which gets called *after* he DynamoBD backend service has been called. What we want to do is move this logic into the `forward_request(..)` method of the same class, and then return the `response` object for each of these `elif` branches.

For a more detailed description of the two proxy methods, please refer to this API documentation:
https://github.com/localstack/localstack/blob/master/localstack/services/generic_proxy.py#L41-L69

Reply:
Thanks for the reference! I just moved the new apis to ```forward_request(..)```